blogs/2016/04/22/edge-debugging-for-vs-code.md

```diff
@@ -0,0 +1,48 @@
+---
+Order: 2
+TOCTitle: Microsoft Edge Debugging for VS Code
```

Comment:
This will be too long for the TOC.  How about just "Microsoft Edge Debugging" or "Edge Browser Debugging".
"Edge Debugging" would be good if you think enough readers will understand.